src/elements/author-profile/src/index.tsx

```diff
@@ -0,0 +1,97 @@
+/** @jsx jsx */
+
+import * as React from 'react'
+import { jsx, Styled } from 'theme-ui'
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+  name: string
+  role: string
+  authorImage?: React.ReactElement
+  authorUrl: string
+  bio: string
+  className?: string
+}
+
+const AuthorProfile: React.FC<Props> = ({
+  name,
+  role,
+  authorImage,
+  authorUrl,
+  bio,
+  className = ''
+}) => {
+  return (
+    <div
+      className={className}
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        flexDirection: 'row',
+        variant: 'authorProfile.main'
+      }}
+    >
+      {authorImage && (
+        <Styled.a
+          href={authorUrl}
+          sx={{
+            variant: 'author.link',
```

Comment:
There's a few variants declared in this file, but I don't see any corresponding theme updates in the PR? Is this intentional?

Reply:
No, I didn't end up using them yet, so could remove them

Reply:
Probably best just to avoid confusion